.github/workflows/on_call_staging_test.yaml

```diff
@@ -42,3 +51,23 @@ jobs:
           
           # Exit with the test result
           exit $exit_code
+
+      - name: Run BDD Tests
```

Comment:
This should use the docker compose items (maybe add a note to readme to favor using compose for workflows steps).
The compose stage should be self contained enough for `docker compose run test` to work, and should be as part of the "Run as service and test" step, after the curl items and before the service is stopped.